src/kokkos_ext/ArborX_DetailsKokkosExtViewHelpers.hpp

```diff
@@ -104,21 +89,14 @@ typename View::non_const_type clone(ExecutionSpace const &space, View const &v)
 
 template <class ExecutionSpace, class View>
 typename View::non_const_type
-cloneWithoutInitializingNorCopying(ExecutionSpace const &space, View const &v,
-                                   std::string const &label)
```

Comment:
Why was the label version removed? Was it not used?

Reply:
It was not used and `Kokkos::create_mirror` doesn't allow passing a new label.